native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json

```diff
@@ -0,0 +1,20 @@
+[
+{
+  "name":"java.lang.String",
+  "methods":[
+    {"name":"lastIndexOf","parameterTypes":["int"] }, 
+    {"name":"substring","parameterTypes":["int"] }
+  ]
```

Comment:
@TharmiganK do we need the above?

Reply:
I think only the resource-config is needed. Will review it before merging 👍